sources/world/Makefile

```diff
@@ -1,12 +1,17 @@
 DATA_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST)))/../../data)
-.DEFAULT_GOAL := $(DATA_DIR)/world_countries_v1.geo.json
+.DEFAULT_GOAL := all
+
+.PHONY: all
+all: ../../data/admin_regions_lvl2_v2.topo.json ../../data/admin_regions_lvl2_v2.geo.json ../../data/admin_regions_lvl2_v1.geo.json ../../data/world_countries_v7.topo.json ../../data/world_countries_v7.geo.json ../../data/world_countries_v1.geo.json # Main target for all resulting datasets
 
 .PHONY: clean
 clean: ## Remove output files
 	rm -f $(DATA_DIR)/admin_regions_lvl2_v1.geo.json \
   $(DATA_DIR)/admin_regions_lvl2_v2.topo.json \
+  $(DATA_DIR)/admin_regions_lvl2_v2.geo.json \
```

Comment:
Do we want to continue to use `$(DATA_DIR)` here? I know we switched back to using relative paths in other places.

Reply:
yeah, the problem is that it cannot be used in the target names or dependencies, so not a lot of chances to benefit from it. I'll remove it.